### PR TITLE
Adding attachment module as a required module by ISS-distro

### PR DESCRIPTION
**Pull request by reagan-meant**
Adding attachment module as a required module by ISS-distro
Issue: https://app.asana.com/0/1164864596604507/1164850277971924

### CODE REVIEW

**Comment by reagan-meant** [comment]
@ssmusoke according to the last comment [here](https://app.asana.com/0/1164864596604507/1164850277971924)...I havent understood the last part about referencing it in distro properties.. Is this [here](https://github.com/METS-Programme/openmrs-module-ugandaemr-iss/blob/master/api/src/main/resources/openmrs-distro.properties) and if yes: could you point me to a template...
Secondly. do i need to add it in the require modules in the config.xml like here [here](https://github.com/METS-Programme/openmrs-module-aijar/blob/master/omod/src/main/resources/config.xml#L42)

**Comment by reagan-meant** [comment]
@ssmusoke the second challenge here is that the bug fixes we want to leverage are in version 2.3.0-SNAPSHOT...When i asked @mksd on irc he adviced that it would be good to ask for a release of version 2.3.0 however it comes with a cost of having to release HTMLFormEntry too....

**Comment by ssmusoke** [comment]
> @ssmusoke the second challenge here is that the bug fixes we want to leverage are in version 2.3.0-SNAPSHOT...When i asked @mksd on irc he adviced that it would be good to ask for a release of version 2.3.0 however it comes with a cost of having to release HTMLFormEntry too....

Releases are not a problem and are not easy, if you ask publicly we can get it done

File: pom.xml [hunk 1]
```diff
@@ -107,6 +108,13 @@
 				<version>${uiframeworkVersion}</version>
 				<scope>provided</scope>
 			</dependency>
+
+			<dependency>
```
Comment (ssmusoke) [inline pom.xml:112]: @reagan-meant You should also add the omod in the distro.properties file (I put that in my comments on the card)

**Comment by reagan-meant** [comment]
Steps to attach patient photo.
1. Do a mvn clean install of this module
2. Deploy module to the server
3.Start the server
4.Add a past visit -> on the patient facing dashboard click attachments located under the different actions
5.Either upload image or take with camera
6. Add caption to the image (probably the patient name)
7 save the form
8Attachment can be seen on the clinician facing dashboard on the attachment widget or in the patient facing dashboard under the visit

File: pom.xml [hunk 2]
```diff
@@ -41,6 +41,7 @@
 		<ugandaemrVersion>3.0.1</ugandaemrVersion>
 		<ugandaemrPocVersion>1.0.4-SNAPSHOT</ugandaemrPocVersion>
```
Comment (ssmusoke) [inline pom.xml]: @reagan-meant increase UgandaEMR POC to 1.0.5-SNAPSHOT which is the current version

File: pom.xml (hunk 1 above)
Comment (reagan-meant) [reply pom.xml:112]: Done

File: pom.xml (hunk 2 above)
Comment (reagan-meant) [reply pom.xml]: Done

**Comment by ssmusoke** [comment]
@reagan-meant you need to add the module to the requires section of the config.xml too

**Comment by reagan-meant** [comment]
Done

**Pull request merged**